src/snake_game/core.cljs

```diff
@@ -1,19 +0,0 @@
-(ns snake-game.core
-  (:require
-   [reagent.core :as reagent :refer [atom]]
-   [re-frame.core :refer [dispatch dispatch-sync]]
-   [snake-game.handlers :as handlers]
-   [snake-game.view :as view]))
-
-;;Dispatch next state event every 150ms
-(defonce snake-moving
```

Comment:
This has always interested me, I prefer to use something I can stop...I guess in this case it does not matter, but for the future, let's keep in mind that all the events can be stopped for some reason